### PR TITLE
new-trip todos will show up on board

### PR DESCRIPTION
**Pull request by supernova-97**
to-dos created in the new trip form will show up on the board.

### CODE REVIEW

File: components/Form/index.js [hunk 1]
```diff
@@ -3,17 +3,17 @@ import styled from "styled-components";
 function Form({ onAddToDo }) {
   function handleSubmit(event) {
     event.preventDefault();
-    const formData = new FormData(event.target);
-    const data = Object.fromEntries(formData);
-    onAddToDo(data);
+
+    onAddToDo(event.target.todo.value);
+    console.log(event.target.todo.value);
```
Comment (mpagels) [inline components/Form/index.js]: here you have a lonely `console.log` that would be merged into production.

File: components/List/index.js [hunk 2]
```diff
@@ -3,10 +3,10 @@ import styled, { css } from "styled-components";
 import { useState } from "react";
 
 function List({ toDos, onDeleteToDo }) {
-  const [isCheckedObject, setIsCheckedObject] = useState({});
+  const [isChecked, setIsChecked] = useState({});
```
Comment (mpagels) [inline components/List/index.js]: When using the name `isChecked` as a variable name, I, as a programmer that coming to the project, would assuming, that here will be a `boolean`. But it is not. Maybe another name would be better, that meets the criteria?

maybe:
- `toDoCheckState` ?

File: components/NewTrip/index.js [hunk 3]
```diff
@@ -1,12 +1,19 @@
...
 
-function NewTrip() {
+function NewTrip({ onAddToDo }) {
+  //"what" input field
+  const [newTripToDos, setNewTripToDos] = useState([]);
+  const [formToDos, setFormToDos] = useState([]);
+  const whatInputRef = useRef(null);
+  //
   const [step, setStep] = useState(true);
+  //input fields
```
Comment (mpagels) [inline components/NewTrip/index.js]: Here are some `comments`. Highly recommend removing them.

File: components/NewTrip/index.js [hunk 4]
```diff
@@ -19,12 +26,39 @@ function NewTrip() {
...
+      data.where === "" ||
+      data.when === 0 ||
+      data.who === ""
+    ) {
       return alert("Missing info!");
     }
+
+    formToDos.map((todo) => {
+      console.log("formToDos", formToDos);
+      console.log("todo.todo", todo.todo);
```
Comment (mpagels) [inline components/NewTrip/index.js]: `console.log` s here. Will noise your `Console` when `formToDos` array has a lot of `todo`s in it.

**Review by mpagels — APPROVED** [review]
Looks very good to me.

I found some minor things:
- `console.log` s that might not need necessary
- a variable name could be better in my opionion.

Pls review and keep up the awesome work!

File: components/List/index.js (hunk 2 above)
Comment (supernova-97) [reply components/List/index.js]: Thanks for the feedback! I keep forgetting about the console.logs

**Review by SarahWett — APPROVED** [review]
lgtm!

**Pull request merged**